datastream-server-api/src/main/java/com/linkedin/datastream/server/DatastreamProducerRecord.java

```diff
@@ -93,6 +95,20 @@ public void setEventsSendTimestamp(long timestamp) {
     _eventsSendTimestamp = Optional.of(timestamp);
   }
 
+  /**
+   * Get timestamp in Epoch-millis for when the events have been audited
```

Comment:
nit: can you rephrase this:
Get saved Kafka header timestamp in Epoch-millis 
or something along those lines?